### PR TITLE
Fix defineUserSignupFields example in migration instructions

### PR DESCRIPTION
**Pull request by Martinsos**
As discussed here https://discord.com/channels/686873244791210014/1205479316226445372/1210232860779479152 .

### CODE REVIEW

File: web/docs/migrate-from-0-11-to-0-12.md [hunk 1]
```diff
@@ -504,10 +504,9 @@ You can follow these steps to migrate to the new auth system (assuming you alrea
     // highlight-start
     export const fields = defineUserSignupFields({
       displayName: async (data) => {
-        if (!data.profile || !data.profile.displayName) {
-          throw new Error('Display name is not available')
-        }
-        return data.profile.displayName
```
Comment (Martinsos) [inline web/docs/migrate-from-0-11-to-0-12.md:510]: This was throwing TS error, so I refactored it to have casting to `any`.

File: web/docs/migrate-from-0-11-to-0-12.md [hunk 2]
```diff
@@ -418,14 +418,14 @@ You can follow these steps to migrate to the new auth system (assuming you alrea
     // highlight-next-line
     export const fields = defineUserSignupFields({
       address: async (data) => {
-        const address = data.address
+        const address = data.address;
```
Comment (infomiho) [inline web/docs/migrate-from-0-11-to-0-12.md:421]: I'd maybe not include the semicolons if we didn't include it in other examples on this page.

File: web/docs/migrate-from-0-11-to-0-12.md (hunk 1 above)
Comment (infomiho) [reply web/docs/migrate-from-0-11-to-0-12.md:510]: I have to admit, I wrote this out of my head and didn't test it. Good catch 👍

**Review by infomiho — APPROVED** [review]

File: web/docs/migrate-from-0-11-to-0-12.md (hunk 1 above)
Comment (infomiho) [reply web/docs/migrate-from-0-11-to-0-12.md:510]: Should we maybe mentioned Zod here ... or more ideally in the `userSignupFields` docs?

**Comment by Martinsos** [comment]
@infomiho feel free to maybe take this one over? YOu will do it better, I just did this quick PR so we have something and we don't forget.

**Pull request merged**